assets/prototype/domain/eventEditor/containers/useConfigProvider.ts

```diff
@@ -0,0 +1,30 @@
+import { useContext } from 'react';
+import { pick } from 'ramda';
+import { ConfigContext } from '../../../infrastructure/services/contextProviders/ConfigProvider';
+
+type configsToBeRetrieved = {
```

Comment:
Have used `type` here, because this isn't yet a case for interface, based on:
![image](https://user-images.githubusercontent.com/1477453/70439826-83191900-1a99-11ea-999e-e574fde1b38a.png)

Reply:
from: https://github.com/typescript-cheatsheets/react-typescript-cheatsheet#types-or-interfaces